tests/test_data/study_valid/manifest.toml

```diff
@@ -8,6 +8,14 @@ file_names = [
 
 [export_config]
 export_list = [
-    "study_valid__table",
+    "study_valid__table"
+]
+count_list = [
+    "study_valid__table2"
+]
+flat_list = [
+    "study_valid__table2"
+]
```

Comment:
This is an interesting set up - do we want to error out if the same table is in two lists? When would you intentionally want that?

Reply:
i don't think you would, no.

I can make this more realistic and add an error - note that this will generate a lot of test churn, sorry in advance.

Reply:
Hah, jokes on you, I just won't read the diff

Reply:
But it does seem nice to catch the duplicate-table-name thing. Seems like a classic copy and paste in a hurry error that folks might actually do.

Reply:
OK, we'll now throw an error in the case.